src/code/z_camera_data.inc.c

```diff
@@ -2565,22 +2565,52 @@ OnePointCsFull D_8011D9F4[] = {
     { 0x12, 0xFF, 0x0000, 0x0001, 0x0000, 60.0f, 1.0f, { -1.0f, -1.0f, -1.0f }, { -1.0f, -1.0f, -1.0f } },
 };
 
-Vec3f D_8011DA6C[] = {
-    { 3050.0f, 700.0f, 0.0f }, { 1755.0f, 3415.0f, -380.0f }, { -3120.0f, 3160.0f, 245.0f }, { 0.0f, -10.0f, 240.0f }
+typedef enum {
+    /* 0 */ CAM_VERTICAL_PLATFORM_FIRE_LOWER_FLOOR,
+    /* 1 */ CAM_VERTICAL_PLATFORM_FIRE_EAST_TOWER,
+    /* 2 */ CAM_VERTICAL_PLATFORM_FIRE_WEST_TOWER,
```

Comment:
is west/east well-defined in OoT?

Reply:
I named this with the concept of the minimap in mind i.e. "west" is the left side of the minimap (where you climb for the hammer) and "east" is the right side of the minimap (where you play scarecrow for 1 of the 6 gold rupee chests), keeping in mind that's for vanilla dungeons, so I didn't want to name it based on that which wouldn't apply for mq dungeons

Reply:
Noticing that all three vertical elevators are different actors, I could also name it based off the actor. Only reason I defaulted to a spatial-descriptive name was the dependence on hard-coded coordinates, but I'm not committed to it

Reply:
Naming it after the actors can only backfire once we Eventually name the actors
These names are fine, I just had the impression/memory OoT was inconsistent about cardinal directions but maybe not? even then, sure